test/integration/controller_test.go

```diff
@@ -575,6 +575,185 @@ func TestProvisionFailure(t *testing.T) {
 	}
 }
 
+func TestBindingFailure(t *testing.T) {
```

Comment:
Can you add a description in the godoc of what this test proves?  Something like:

```go
// TestBindingFailure tests that a binding gets a failure condition when
// the broker returns a failure response for a bind operation.
```